duckduckgo/iOS
# Add site rating to content blocker

Reviewer: Chris
Asana: https://app.asana.com/0/361428290920652/354154035676009

**Description**:
- adds a site rating to the omnibar content blocker button
- adds a https connection indicator to the content blocker panel

**Steps to test this PR**:
TEST ONE
1. Navigate to a https website
2. Ensure that the content blocker panel displays this as a secure connection

TEST TWO
1. Navigate to a http website
2. Ensure that the content blocker panel displays this as an unsecured connection

TEST THREE
1. Navigate to a http page with numerous trackers and ensure that the rating is bad e.g the daily mail is a C (it will be D in the future when we support more trackers)
2. Navigate to a https with few trackers and ensure the rating is good e.g our website is an A rating
3. Ensure the rating is reset when moving from one page to another

###### Reviewer Checklist:


###### PR DRI Checklist:
